hypervisor/Cargo.toml

```diff
@@ -26,7 +26,7 @@ vfio-ioctls = { git = "https://github.com/rust-vmm/vfio", branch = "main", defau
 vm-memory = { version = "0.11.0", features = ["backend-mmap", "backend-atomic"] }
 vmm-sys-util = { version = "0.11.0", features = ["with-serde"] }
 
-[target.'cfg(target_arch = "x86_64")'.dependencies.iced-x86]
+[target.'cfg(all(target_arch = "x86_64", target_os = "windows"))'.dependencies.iced-x86]
```

Comment:
MSHV runs on Linux - this is covered by the "mshv" feature

Reply:
> MSHV runs on Linux

TIL, cool.

> - this is covered by the "mshv" feature

Yes. The intent was to skip downloading the `iced-x86` and its dependencies, when `mshv` is _not_ the hypervisor backend.